packages/contracts/src/framework/plugin/repo/PluginRepo.sol

```diff
@@ -129,6 +114,22 @@ contract PluginRepo is
         _grant(address(this), initialOwner, UPGRADE_REPO_PERMISSION_ID);
     }
 
+    /// @notice Initializes the pluginRepo after an upgrade from a previous protocol version.
+    /// @param _previousProtocolVersion The semantic protocol version number of the previous DAO implementation contract this upgrade is transitioning from.
+    /// @param _initData The initialization data to be passed to via `upgradeToAndCall` (see [ERC-1967](https://docs.openzeppelin.com/contracts/4.x/api/proxy#ERC1967Upgrade)).
+    /// @dev This function is a placeholder until we require reinitialization.
+    function initializeFrom(
+        uint8[3] calldata _previousProtocolVersion,
+        bytes calldata _initData
+    ) external reinitializer(2) {
+        // Silences the unused function parameter warning.
+        _previousProtocolVersion;
+        _initData;
+        
```

Comment:
Test that this is correct and dont need reinitialization between v1.0.0 and v1.3.0